src/openklant/conf/api.py

```diff
@@ -4,8 +4,8 @@
 # Remove the reference - we don't have a single API version.
 del API_VERSION  # noqa
 
-KLANTINTERACTIES_API_VERSION = "0.1.2"
-CONTACTGEGEVENS_API_VERSION = "1.1.1"
+KLANTINTERACTIES_API_VERSION = "0.1.3"
+CONTACTGEGEVENS_API_VERSION = "1.1.2"
```

Comment:
Not sure what changed for Contactgegevens API, just phone validation? If that's the case, then 1.1.2 is fine
```suggestion
KLANTINTERACTIES_API_VERSION = "0.2.0"
CONTACTGEGEVENS_API_VERSION = "1.1.2"
```

Reply:
@stevenbal actually, the phone number validation is only used in `klantinteracties`, so I change back to the previous API version